compiler-rt/test/asan/TestCases/Posix/wait4.cpp

```diff
@@ -4,21 +4,13 @@
 // RUN: %clangxx_asan -DWAIT4_RUSAGE -O0 %s -o %t && not %run %t 2>&1 | FileCheck %s
 // RUN: %clangxx_asan -DWAIT4_RUSAGE -O3 %s -o %t && not %run %t 2>&1 | FileCheck %s
 
-// XFAIL: android
 // UNSUPPORTED: darwin
 
 #include <assert.h>
 #include <sys/wait.h>
 #include <unistd.h>
 
 int main(int argc, char **argv) {
-  // This test passes on some versions of Android NDK and fails on other.
-  // https://code.google.com/p/memory-sanitizer/issues/detail?id=64
```

Comment:
looks like https://github.com/google/sanitizers/issues/567 was the correct bug, and, yes, this was only for API < 18, so this is definitely obsolete.